cli/tsc/dts/lib.deno_webgpu.d.ts

```diff
@@ -216,6 +216,12 @@ declare type GPUBufferUsageFlags = number;
 /** @category WebGPU */
 declare type GPUFlagsConstant = number;
 
+/** @category WebGPU */
+declare type GPUImageCopyExternalImageSource =
+  | ImageBitmap
+  | ImageData
+  | GPUCanvasContext;
```

Comment:
I picked this union type from here.

https://www.w3.org/TR/2024/WD-webgpu-20240409/#typedefdef-gpuimagecopyexternalimagesource